Tests/MapboxNavigationTests/CarPlayManagerSpec.swift

```diff
@@ -173,7 +173,7 @@ class CarPlayManagerSpec: QuickSpec {
                 }
             }
         }
-        
+#if arch(x86_64) && canImport(Darwin)
```

Comment:
This test contains a `throwAssertion()` call which is not supported on M1 macs.
https://github.com/Quick/Nimble/blob/e4aa2014de13b90b06eaf183c67be3e390e927da/Sources/Nimble/Matchers/ThrowAssertion.swift#L46